### ISSUE TITLE
Configuration directives rely on absolute paths

### ISSUE THREAD

**Issue by nmav** [ANSSI-FR/caml-crush#7]
The configuration directives for example in filter.conf rely on absolute paths to obtain the used library. That means that configuration files cannot be shared by multiple platforms, and more importantly multi-arch distributions cannot possibly use it. For example the softhsm library may be available on the following two different paths depending on the architecture:
modules = [("softhsm", "/usr/lib64/pkcs11/libsofthsm.so")]
modules = [("softhsm", "/usr/lib/pkcs11/libsofthsm.so")]

I think the best would be to use the --libdir configure option and then allow relative library paths over that. That way there can be a 32-bit pkcs11proxyd and 64-bit pkcs11proxyd installed in the same system and share the same configuration file.

**Comment by tc-anssi** [comment]
Indeed, I will investigate and check if OCamlnet Netplex configuration parser can use relative path (last time I check it did not).
